### PR TITLE
document typedef file system-elements section

### PR DESCRIPTION
**Pull request by CarolynRountree**
Fixes Jira #360

Add documentation for the system-elements section of the typedef file.

### CODE REVIEW

File: documentation/2.0/content/userguide/tools-config/domain_def.md [hunk 1]
```diff
@@ -185,3 +185,134 @@ weight: 2
  ```
 
  The `targeting` attribute is not valid for WebLogic Server versions 12.2.1 and up.
+
+#### Using the typedef file to exclude template installed resources from the model and archive
+
+ The Discover tool attempts to provide a sparse model by doing such things as not including attributes that are default values. It is important to not install template resources through the model. First, the templates will install the same resources in the target domain . Second, if you install resources from a on-premise domain into a target domain that is a later WebLogic version, it could cause conflicts.
+ The `system-elements` section of the typedef file is used to assist the Discover tool. The `system-elements` section is a list that tells discover what to exclude from the model and archive files.
```
Comment (rosemarymarano) [inline documentation/2.0/content/userguide/tools-config/domain_def.md:192]: Discover tool -> Discover Domain Tool
section is a list -> comprises a list
that tells discover -> that tells the tool

File: documentation/2.0/content/userguide/tools-config/domain_def.md [hunk 2]
```diff
@@ -185,3 +185,134 @@ weight: 2
...
+|`coherence-clusters` |
+| `datasources` |
+| `file-stores`|
+| `jms` |
+| `jms-servers` |
+| `shared-libraries` |
+| `startup-classes` |
+| `wldf` |
+
+The following example is the `system-elements` list in the JRF.json typedef file.
```
Comment (rosemarymarano) [inline documentation/2.0/content/userguide/tools-config/domain_def.md:210]: the JRF.json typedef file. -> the `JRF.json` (code font) typedef file.

**Review by rosemarymarano — COMMENTED** [review]
Few edits

**Review by rosemarymarano — APPROVED** [review]
LGTM

**Review by robertpatrick — APPROVED** [review]

**Review by rakillen — APPROVED** [review]

**Review by ddsharpe — APPROVED** [review]

**Pull request merged**